packages/relay/src/lib/errors/SDKClientError.ts

```diff
@@ -71,6 +71,10 @@ export class SDKClientError extends Error {
     return this.statusCode === Status.Unknown._code && this.message?.includes('timeout exceeded');
   }
 
+  public isConnectionDropped(): boolean {
+    return this.statusCode === Status.Unknown._code && this.message?.includes('Connection dropped');
```

Comment:
Q: which tool results in this error message? Is it axios?
Might be worth a comment

Reply:
No, SDK returns this error.